index.js

```diff
@@ -1,25 +1,23 @@
-'use strict';
-
 import {
   DeviceEventEmitter,
   NativeModules,
   Platform,
 } from 'react-native';
 
-const nativeModule = NativeModules.RNProximity;
+const { RNProximity } = NativeModules;
```

Comment:
Need to also change the Android if to use `RNProximity` instead of `nativeModule`